### PR TITLE
NTGR-612: Update the order of the auto issuance logs

### PR DESCRIPTION
**Pull request by kei178**
This PR adds `$options` to `Accredible_Learndash_Model::get_results` in order to update the order of the auto issuance logs list page.

### CODE REVIEW

File: tests/includes/models/test-class-accredible-learndash-model-auto-issuance-log.php [hunk 1]
```diff
@@ -65,6 +65,19 @@ public function test_get_results() {
 		$results = Accredible_Learndash_Model_Auto_Issuance_Log::get_results( '', $limit, $offset );
 		$this->assertCount( 1, $results );
 		$this->assertEquals( $data2_id, $results[0]->id );
+
+		// With $limit & $offset.
```
Comment (Ndibui) [inline tests/includes/models/test-class-accredible-learndash-model-auto-issuance-log.php:69]: Duplicate test case.

**Review by Ndibui — COMMENTED** [review]
Looks good. Minor correction on the duplicate test case.

File: tests/includes/models/test-class-accredible-learndash-model-auto-issuance-log.php (hunk 1 above)
Comment (kei178) [reply tests/includes/models/test-class-accredible-learndash-model-auto-issuance-log.php]: Thanks for the catch! I removed the duplicate test case.

**Review by octaviorequena — APPROVED** [review]
LGTM!

**Review by Ndibui — APPROVED** [review]

**Pull request merged**